test/smoke/testApps/CoreAndFilter/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/CoreAndFilterTests.java

```diff
@@ -291,7 +291,7 @@ public void testTrackPageView() throws Exception {
 
         assertNotNull(pv2);
         assertEquals(new Duration(123456), pv2.getDuration());
-        assertEquals("2010-10-10T00:00:00.000+0000", pvdEnvelope2.getTime());
+        assertEquals("2010-10-10T00:00:00Z", pvdEnvelope2.getTime());
```

Comment:
what changed? or did we miss this failure previously?

Reply:
After latest otel update on the aad branch, I encountered this smoketest failing, hence the change.

Reply:
I'm not sure this is the correct breeze format..

Reply:
nevermind, this is good 👍